treeherder/services/pulse/consumers.py

```diff
@@ -199,6 +207,8 @@ def bindings(self):
             rv += GITHUB_PUSH_BINDINGS
         if self.source.get('tasks'):
             rv += TASKCLUSTER_TASK_BINDINGS
+        if self.source.get('mozci-classification'):
+            rv += MOZCI_CLASSIFICATION_BINDINGS
```

Comment:
Are these bindings for the firefox-ci cluster or community-cluster?

Reply:
Something else you might want to look at for troubleshooting is the PROJECTS_TO_INGEST and how this env variable would affect what you're consuming: https://github.com/mozilla/treeherder/blob/master/treeherder/etl/taskcluster_pulse/handler.py#L17

Also, have you been able to get this data ingesting locally? This might help with troubleshooting: https://treeherder.readthedocs.io/pulseload.html

Reply:
Thx!